settings/language-puppet.cson

```diff
@@ -19,3 +19,11 @@
       'source'
       'unless'
     ]
+    'increaseIndentPattern': '(?x)
```

Comment:
Gotcha. I didn't know there were triple quotes in CSON.

I also simplified the pattern. (See latest commits.)